content/foundations/icons/octovisuals.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Octovisuals
+source: https://github.com/github/octovisuals
+---
+
+import {Box, Text, Link} from '@primer/react'
+import {LinkExternalIcon, DownloadIcon} from '@primer/octicons-react'
+import OctovisualsBanner from './images/octovisuals.png'
+import './octovisuals.css'
+
+### Octovisuals are GitHub's internal set of logos and visuals for brand and marketing usage.
+
+<br />
+
+GitHub's brand is visually expressed in a multitude of ways depending on the project, audience, and more. We use various brand assets, such as Octovisuals, to represent differnt aspects of GitHub. **Octovisuals are intended for internal use only and can only be accessed by Github staff.**
+
+<p>
+  <a href="https://octovisuals.github.com/" target="_blank">
+    See the full set of Octovisuals here
+  </a>
+  &nbsp;
+  <LinkExternalIcon size={16} fill="#0969da" />
+  &nbsp;&nbsp;<span style="color:#6E7781">(GitHub staff only)</span>
+</p>
+
+<br />
+
+<img src={OctovisualsBanner} style="width:100%" alt="Octovisuals" />
```

Comment:
You can use Box to avoid the inline style here..

```jsx
<Box as="img" src={OctovisualsBanner} sx={{width:"100%"}} alt="Various Octicons in a grid arrangement" />
```